bin/pygrb/pycbc_pygrb_minifollowups

```diff
@@ -94,11 +87,12 @@ def make_timeseries_plot(workflow, trig_file, snr_type, central_time,
     node.add_opt('--x-lims='+x_lims)
     # Plot title
     if ifo is not None:
-        title_str = "'%s SNR at %.3f (s)'" %(ifo, central_time)
-        node.add_opt('--central-time', central_time)
+        title_str = "'%s SNR at %.3f (s)'" % (ifo, central_time)
```

Comment:
```suggestion
        title_str = f"'{ifo} SNR at {central_time:.3f} (s)'"
```
Are the double quotes intended? If not, remove those too.

Reply:
They are.  This way the .sh file reports, e.g.,  `--plot-title 'Coherent SNR at 1185259043.073 (s)'` instead of `--plot-title Coherent SNR at 1185259043.073 (s)`, which fails.